charts/csm-authorization-v2.0/templates/storage-service.yaml

```diff
@@ -145,18 +145,13 @@ spec:
           readOnly: true
         {{- end }}
         {{- end }}
-        {{- if .Values.storageSystemCredentials.secrets }}
-        {{- range .Values.storageSystemCredentials.secrets }}
+        {{- if .Values.global.storageSystemCredentials.secrets }}
+        {{- range .Values.global.storageSystemCredentials.secrets }}
         - name: storage-system-secrets-{{ . }}
           mountPath: /etc/csm-authorization/{{ . }}
           readOnly: true
         {{- end }}
         {{- end }}
-        {{- if .Values.redis.redisSecretProviderClass.redisSecretName }}

```

Comment:
May I know why is this removed? Doesn't storage service need this?

Reply:
The SPC values are already being mounted in the storage service.

Reply:
If they use the same SPC for redis creds, it would be a duplicate entry which helm charts won't allow. However, if they create a different SPC to hold redis creds, we will need the mount path and volume in the storage service. Need to think about this one more.